html/janus.js

```diff
@@ -1283,6 +1289,9 @@ function Janus(gatewayCallbacks) {
 						    'optional': []
 						};
 					}
+					if(typeof media.video === 'object') {
+						videoSupport = media.video;
```

Comment:
Similar as the audio custom constraint definition. If we know the deviceId - we don't need for janus to recognize it